src/wp-content/themes/twentytwentyfive/patterns/banner-intro-image.php

```diff
@@ -39,7 +39,7 @@
 			<div class="wp-block-buttons">
 				<!-- wp:button -->
 				<div class="wp-block-button">
-					<a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Learn More', 'Button text of intro section.', 'twentytwentyfive' ); ?></a>
+					<a class="wp-block-button__link wp-element-button"><?php esc_html_e( 'Learn More', 'twentytwentyfive' ); ?></a>
```

Comment:
Using 'Learn More' in the `banner-intro-image` and `hero-full-width-image` patterns would make them consistent with each other, but the existing string in `cta-centered-heading` has a lowercase m:

https://github.com/WordPress/wordpress-develop/blob/626d9af280566d008ccf501952f119fb0ce61662/src/wp-content/themes/twentytwentyfive/patterns/cta-centered-heading.php#L28

Reply:
Yes let's use sentence case here :)